.bazelignore

```diff
@@ -0,0 +1 @@
+tool/ide/test
```

Comment:
Bazel ignores the existence of WORKSPACE and treats the directory as a bazel package, which leads to circular dependency issues